handlers/replace.js

```diff
@@ -8,7 +8,7 @@ const LOG = log4js.getLogger('replace');
  */
 module.exports = {
   name: 'replace',
-  trigger: '.r',
+  trigger: '.r ',
```

Comment:
Had to add a space to this command because using `.roll` would just trigger the replace command

Reply:
I'll figure out a better way to do this :0 hmmm

Reply:
suggestion:

Take the part of the handler that checks to see if it starts with a string and instead split the string on spaces and check the first index?

```js
...
    if (_.isString(trigger)) {
      return message.content.startsWith(trigger);
    }
...
```

becomes

```js
    if (_.isString(trigger)) {
      return message.content.split(delimiter)[0] === trigger; ;
    }
```

in general I think adding excess spaces to input commands becomes a little silly and it's more indicative of the input handling being a problem, so imo the `.r` command shouldn't have to change, y'know? 

Anyways thems my thoughts.

Reply:
Yeah this is a good call, good idea